src/components/mainPage/ResultList.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="films-list">
+    <div class="film-card" v-for="film in films" v-bind:key="film.title">
+      <router-link to="/film-details"
+        ><FilmCard
+          :imageSrc="film.image"
+          :title="film.title"
+          :year="film.year"
+          :genre="film.genre"
+      /></router-link>
+    </div>
+  </div>
+</template>
+
+<script>
+import FilmCard from "@/components/FilmCard";
+export default {
+  components: {
+    FilmCard
+  },
+  data() {
+    return {
+      films: [
+        {
```

Comment:
As explained in one of the above comments, it's way more convenient to keep data separately from components and reuse it throughout the app.

Reply:
Yes, I agree, but can I leave this without changes in this PR? In the next PR I moved all the data to the store

Reply:
Sure